.github/workflows/ci.yml

```diff
@@ -106,7 +106,7 @@ jobs:
         uses: zgosalvez/github-actions-report-lcov@v2
         with:
           coverage-files: ./lcov.info
-          minimum-coverage: 70 # Set coverage threshold.
+          minimum-coverage: 67 # Set coverage threshold.
```

Comment:
We can bump this up to 92 or so now, I think!

Reply:
Done in 12b33a6